### PR TITLE
style(react-component-activity-share-file): add overflow for file name

### LINKED ISSUE #579: Long file names go beyond the grey wrapper
Screenshot:

sparkissue1

### PR DESCRIPTION
**Pull request by adamweeks**
fixes #579

### CODE REVIEW

**Comment by adamweeks** [comment]
Deploy preview for *hopeful-hamilton-4090b9* ready!

Built with commit 46416d08b49c23ef77b6923ec47c679bf7371134

https://deploy-preview-623--hopeful-hamilton-4090b9.netlify.com

File: packages/node_modules/@ciscospark/react-component-activity-share-file/src/styles.css [hunk 1]
```diff
@@ -32,6 +32,11 @@
   flex: 1 1 auto;
 }
 
+.file-name {
+  max-height: 35px;
+  overflow: hidden;
```
Comment (bzang) [inline packages/node_modules/@ciscospark/react-component-activity-share-file/src/styles.css]: Just hiding the overflow isn't to spec. It needs to have ellipsis like this: 
screen shot 2018-01-16 at 2 42 08 pm

**Comment by adamweeks** [comment]
@bzang fixed

**Comment by bzang** [comment]
Still a bit off.... the download icon is now outside of the box...
screen shot 2018-01-19 at 10 50 21 am

**Comment by adamweeks** [comment]
img

**Comment by adamweeks** [comment]
Released in 0.1.245

**Pull request merged**